.travis.yml

```diff
@@ -8,7 +8,8 @@ node_js:
 
 branches:
   only:
-    - master
+    #- master
+    - fiqare-improvements-1
```

Comment:
This should be reverted (so .travis.yml ends with no change in this PR at the end)

Reply:
I'm going to revert the changes, sorry.

Reply:
Fixed in 08201e79ae90920e0f9707b01629f3f281c7d4ad